clients/src/main/java/org/apache/kafka/common/errors/ConcurrentTransactionsException.java

```diff
@@ -16,6 +16,12 @@
  */
 package org.apache.kafka.common.errors;
 
+/**
+ * This exception indicates that a concurrent exception has occurred.
+ *
+ * The producer attempted to update a transaction while
```

Comment:
I think `concurrent exception has occurred` is not clear enough. Could we just say: `This exception indicates that the producer attempted to update a transaction while another concurrent operation on the same transaction was ongoing.` ?